README.md

```diff
@@ -1,11 +1,61 @@
-# KEVM Ganache-CLI fork
+KEVM Ganache-CLI Fork
+=====================
+
+Installing KEVM
+---------------
 
-## Installing KEVM
 In order to run `kevm-ganache-cli`, you will need to install `kevm-vm`.
-We currently support [Ubuntu](#Ubuntu), [Arch](#Arch), and [MacOS](#MacOS).
+We currently support Ubuntu Bionic (18.04) and Arch Linux (MacOS coming soon).
+Here a specific GitHub release is used, feel free to replace the URL with a newer release from <https://github.com/kframework/evm-semantics/releases>.
+
+### Ubuntu Bionic (18.04) - fresh VM
+
+```sh
+sudo apt install nodejs npm
+curl --location 'https://github.com/kframework/evm-semantics/releases/download/v1.0.0-9ae34f5/kevm_1.0.0_amd64.deb' --output kevm_1.0.0_amd64.deb
+sudo apt install ./kevm_1.0.0_amd64.deb
+```
+
+### Arch Linux - fresh VM
+
+```sh
+sudo pacman -S nodejs npm git python2 make gcc
+curl --location 'https://github.com/kframework/evm-semantics/releases/download/v1.0.0-9ae34f5/kevm-git-1.0.0-1-x86_64.pkg.tar.xz' --output kevm-git-1.0.0-1-x86_64.pkg.tar.xz
+sudo pacman -U kevm-git-1.0.0-1-x86_64.pkg.tar.xz
+```
+
+Install `npx`
+-------------
+
+Install the `npx` executable globally with:
+
+```sh
+sudo npm install -g npx
+```
+
+Install KEVM Ganache CLI and Truffle
+------------------------------------
+
+This downloads and installs the KEVM variant of `ganache-cli` and the test-runner `truffle`.
+
+```sh
+npm install kevm-ganache-cli truffle
```

Comment:
This would install these packages in the current directory, under `node_modules`. I'm not sure this is what we want. We probably either want to globally install them, or, rather, to not install `truffle` (since `openzeppelin`'s `npm install` already does that anyway). We also do not need to install `kevm-ganache-cli` in a random folder, but we can just do `npx kevm-ganache-cli` and it should work.

So, the suggestion is to remove this section entirely (from "Install KEVM Ganache CLI and Truffle" up to "Run some Tests".

Reply:
Done